frontend/src/components/small components/TimeTicker.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+
+export const TimeTicker = () => {
+  const [timeState, setTimeState] = useState();
+
+  useEffect(() => {
+    setInterval(() => {
+      const date = new Date();
+      setTimeState(date.toLocaleTimeString());
+    }, 1000);
+  }, []);
```

Comment:
You should add `clearInterval` function when component unmounts to clear out browser resources :

```
  useEffect(() => {
    const interval = setInterval(() => {
      const date = new Date();
      setTimeState(date.toLocaleTimeString());
    }, 1000);
    
    return () => {
      clearInterval(interval);
    }
  }, []);
```